extensions/blocks/google-calendar/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import { extractAttributesFromIframe, URL_REGEX, IFRAME_REGEX } from './utils';
+import './editor.scss';
+import icon from './icon';
+
+export const name = 'google-calendar';
+export const title = __( 'Google Calendar', 'jetpack' );
+
+export const settings = {
+	title,
+	description: __( 'Embed a Google Calendar', 'jetpack' ),
+	keywords: [
+		_x( 'events', 'block search term', 'jetpack' ),
+		_x( 'dates', 'block search term', 'jetpack' ),
+	],
+	icon,
+	category: 'jetpack',
+	supports: {
+		align: true,
+		alignWide: false,
```

Comment:
Would it be an option to support wide alignment at some point? I think it would be nice since calendars are much more usable when they're larger imo.

Reply:
Not sure we we disabled this initially as it seems to work fine out of the box for me, have enabled it.